plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/HiveMetastoreModule.java

```diff
@@ -47,10 +48,7 @@ protected void setup(Binder binder)
             bindMetastoreModule("thrift", new ThriftMetastoreModule());
             bindMetastoreModule("file", new FileMetastoreModule());
             bindMetastoreModule("glue", new GlueMetastoreModule());
-            // Load Alluxio metastore support through reflection. This makes Alluxio effectively an optional dependency
```

Comment:
> Revert "Defer initialization of AlluxioMetastoreModule"

> This only allow to bypass the vulnerability by removing jars. We should
not recommend users to play with jars of plugins. It is very risky
operation. Instead we should fix the Alluxio vulnerabilities.

Being able to tamper with the tarball (remove Alluxio) is still better than nothing.
In my eyes, the Alluxio integration does not look maintained and should perhaps be dropped. Until then, it's probably beneficial to unlock shipping Trino without Alluxio. Of course, such use-case is not recommended, nor supported, but we shouldn't actively work to make it impossible.